backend/controller/sql/queries.sql

```diff
@@ -364,12 +364,12 @@ VALUES ((SELECT id FROM deployments WHERE name = $1 LIMIT 1), $2, $3, $4, $5);
 
 -- name: GetIngressRoutes :many
 -- Get the runner endpoints corresponding to the given ingress route.
-SELECT r.key AS runner_key, endpoint, ir.module, ir.verb
+SELECT r.key AS runner_key, endpoint, ir.path, ir.module, ir.verb
 FROM ingress_routes ir
          INNER JOIN runners r ON ir.deployment_id = r.deployment_id
 WHERE r.state = 'assigned'
   AND ir.method = $1
-  AND ir.path = $2;
+  AND ir.path LIKE CONCAT(sqlc.arg('path')::TEXT, '%');
```

Comment:
Not sure if this is good, or a bit too hacky. Open to other ideas here. Basically, I need to find routes that start with the path I'm looking for and then filter it down later based on the actual request that was made on the client.

Reply:
I don't think there's a good solution that uses the DB (or at least, not one I can think of).  I think we'll just need to load the entire ingress table into memory periodically, and do the matching in code.